README.md

```diff
@@ -1 +1,192 @@
-# to_do_list
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 To-Do list <a name="about-project"></a>
+
+- This website shows a sneak peek into the projects I have developed, showcasing my skills in HTML, CSS, javascript and webpack. With a passion for web development, I have honed my expertise in these technologies to create stunning and functional websites.
+
+## 🛠 Built With <a name="built-with"></a>
+
+- HTML
+- CSS
+- linter
+- webpack
+- HTML validation
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo
+
+- You can see the demo [Here](coming soon)
+
+## Key Features
+
+- Use webpack for older browser responsivness
+- Use the semantics elements💯
+-  Use the linter for fixing errors .🚀
+- Use the flexbox to place the items 😃👍
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+
+Example command:
+
+```sh
+1.use a browser
+2.use cable internet
+```
+ 
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+Example commands:
+
+```sh
+  
+  git clone git@github.com:albertkantwi/to_do_list.git
+  cd to_do_list
+
+
+```
+
+
+### Install
+
+Install this project with:
+
+
+Example command:
+
+```sh
+  npm install
+```
+
+
+### Usage
+
+To run the project, execute the following command:
+
+
+Example command:
+
+```sh
+- Open the index with live server.
+- Or if you don't have a live server extension open the index in the browser by clicking on the file .
+```
```

Comment:
- [ ] To run the webpack, please add here the command 'npm run dev' to start the project on the developer mode as per the requirement.